routes/per_capita_consumption.py

```diff
@@ -35,7 +36,7 @@ def get_moving_average_annualized(meter_id):
         return None
 
 
-@PerCapitaConsumption.route('/per-capita-consumption', methods=['GET'])
+@ShowPerCapitaConsumption.route('/per-capita-consumption', methods=['GET'])
```

Comment:
See above - there is no harm in leaving the route name as it is, but changing the Blueprint name internally to a more telling one.